Utilities/rXml.cpp

```diff
@@ -18,37 +18,50 @@ rXmlNode::rXmlNode(void *ptr)
 	handle = ptr;
 }
 
+// These functions go unused.
+/*
 rXmlNode::rXmlNode(const rXmlNode& other)
 {
 	ownPtr = true;
-	handle = reinterpret_cast<void *>(new wxXmlNode(*reinterpret_cast<wxXmlNode*>(other.handle)));
+	handle = reinterpret_cast<void *>(new pugi::xml_node(*reinterpret_cast<pugi::xml_node*>(other.handle)));
 }
 
 rXmlNode &rXmlNode::operator=(const rXmlNode& other)
 {
 	if (ownPtr)
 	{
-		delete reinterpret_cast<wxXmlNode*>(handle);
+		delete reinterpret_cast<pugi::xml_node*>(handle);
 	}
-	handle = reinterpret_cast<void *>(new wxXmlNode(*reinterpret_cast<wxXmlNode*>(other.handle)));
+	handle = reinterpret_cast<void *>(new pugi::xml_node(*reinterpret_cast<pugi::xml_node*>(other.handle)));
 	ownPtr = true;
 	return *this;
 }
+*/
 
 rXmlNode::~rXmlNode()
 {
 	if (ownPtr)
 	{
-		delete reinterpret_cast<wxXmlNode*>(handle);
+		delete reinterpret_cast<pugi::xml_node*>(handle);
```

Comment:
yeah, you can store the type. It was just a `void*` because I was too lazy/stupid to forward declare the class (because I wanted it usable in the core without linking to wx).